inception-log/src/main/java/de/tudarmstadt/ukp/inception/log/EventRepositoryImpl.java

```diff
@@ -43,4 +46,24 @@ public void create(LoggedEvent aEvent)
         log.info("{}", aEvent);
         entityManager.persist(aEvent);
     }
+
+    @Override
+    @Transactional
+    public List<LoggedEvent> listLoggedEvents(Project aProject, String aUsername, String eventType,
```

Comment:
`eventType` and `size` miss the initial `a`.

Reply:
Done.